test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/jms/KieServerS2iAmqOptaplannerIntegrationTest.java

```diff
@@ -56,7 +55,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(Parameterized.class)
-@Category({ApbNotSupported.class, OperatorNotSupported.class})
+@Category({ApbNotSupported.class})
```

Comment:
I think there are 2 more test classes from where should be removed OperatorNotSupported category.

Reply:
Even that PR is related to RHDM only, test of RHPAM should be skipped as there is JBPMOnly category.

Reply:
jBPM tests are skipped automatically through UnsupportedOperationException check